### ISSUE TITLE
echo command not explained on first use in pipes and filters

### ISSUE THREAD

**Issue by injaci** [swcarpentry/shell-novice#877]
Please delete the text below before submitting your contribution. 

---
We have seen the use of >, but there is a similar operator >> which works slightly differently. By using the echo command to print strings, test the commands below to reveal the difference between the two operators:

Instead the echo command is explained in the next loops section. This explanation can be added where the command is first used.

The first, echo, just prints its command-line arguments to standard output. For example:
$ echo hello there
prints:
hello there
---

**Comment by Bitmapped** [comment]
I don't believe a change is needed. The section in question already says what `echo` does: 
> By using the `echo` command to print strings,...

The thing that might not be explicitly clear right now is where `echo`'s output goes by default (the screen). I think most users would figure that out from context since this entire unit deals with redirecting output from the screen to elsewhere.

I tried thinking of other ways of phrasing the text to clarify that `echo`'s output defaults to the screen, but since both uses of `echo` here involve redirecting its output, everything I came up with ended up being needlessly complicated.

My suggestion would be to leave things as they are. If others are concerned about using `echo` here, perhaps it would be best to write the examples to use a different command instead of `echo` for generating the output.
